back/transcendence_django/back_game/game_arena/player_manager.py

```diff
@@ -167,8 +173,16 @@ def __change_player_status(self, user_id: int, status: PlayerStatus):
     def __finish_player(self, user_id: int):
         self.__change_player_status(user_id, PlayerStatus(OVER))
 
-    def __fill_player_specs(self, players_specs: dict[str, int]):
+    def __fill_player_specs(self, players_specs: dict[str, Any]):
         self.nb_players = players_specs[NB_PLAYERS]
+        self.nb_humans = (
+            players_specs["options"]["human_players"]
```

Comment:
define/change and use dict keys:

OPTIONS
HUMAN_OPPONENTS_LOCAL
HUMAN_OPPONENTS_ONLINE
AI_OPPONENTS_LOCAL
AI_OPPONENTS_ONLINE

(change in front too)

Reply:
Front does not have dict keys.
That aside, all g.